test/ignore_test.go.ignore

```diff
@@ -0,0 +1,122 @@
+// +build e2e
+
+/*
+ Copyright 2021 The Tekton Authors
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package test
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/test/parse"
+	corev1 "k8s.io/api/core/v1"
+	"knative.dev/pkg/apis"
+	duckv1beta1 "knative.dev/pkg/apis/duck/v1beta1"
+	"testing"
+
+	"github.com/ghodss/yaml"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestToYAML(t *testing.T) {
+	helloworldResource := &v1alpha1.PipelineRun{
+		ObjectMeta: metav1.ObjectMeta{Name: "bar"},
+		Status: v1alpha1.PipelineRunStatus{Status: duckv1beta1.Status{
+			Conditions: []apis.Condition{{Type: apis.ConditionSucceeded, Status: corev1.ConditionTrue}},
+		}},
+	}
+	ty, err := yaml.Marshal(helloworldResource)
+	require.NoError(t, err)
+	fmt.Printf("yaml:\n%s\n", string(ty))
+}
+
+func TestCompBetaAlpha(t *testing.T) {
```

Comment:
I'm a bit confused about what this test is for; this almost looks like a test for pipelineresources? why is it called ignore_test? I'm not sure the changes you're introducing need a new e2e test.

Reply:
Hahaha, this is detritus from who knows when that was still sitting in my local clone and somehow inadvertently got added. Nuking it now!